conf.d/rewrite.conf

```
@@ -30,4 +12,31 @@ rewrite /experimental/order-summary-template/(.*) https://pages.github.ibm.com/C
 rewrite /experimental/(.*) /community/patterns/\$1 permanent;
 
 # Community index
-rewrite /community/components /community/component-index permanent;
\ No newline at end of file
+rewrite /community/components /community/component-index permanent;
+
+# IA Updates 8/2020
+# All about Carbon
+rewrite /get-started/about-carbon /all-about-carbon/what-is-carbon permanent;
+rewrite /contributing/governance /all-about-carbon/how-we-work permanent;
+
+# Designing
+rewrite /get-started/design/(.*) /designing/get-started/kits/\$1 permanent;
+
+# Contributions rewrites
+rewrite /how-to-contribute/(.*) /contributing/\$1 permanent;
+
+# What's happening
+rewrite /updates/whats-new /whats-happening/changelog-and-roadmap permanent;
+rewrite /updates/work-in-progress /whats-happening/work-in-progress permanent;
+
+# Help
+rewrite /updates/migration-guide/(.*) /help/migration-guide/\$1 permanent;
+rewrite /help/support /help/contact-us permanent;
+rewrite /help/faq /help/faqs permanent;
+
+# Developing
+rewrite /get-started/develop/(.*) /developing/frameworks/\$1 permanent;
```

```suggestion
rewrite /get-started/develop(.*) /developing/frameworks/\$1 permanent;
```

We  should prob be consistent with how we handle slashes with these.

---

don't we need that slash there, so it matches any of the pages that hang off of /develop and redirects them to pages off of /frameworks?